src/unistd.rs

```diff
@@ -1228,11 +1228,14 @@ pub fn isatty(fd: RawFd) -> Result<bool> {
     }
 }
 
-/// Flags for `linkat` function.
-#[derive(Clone, Copy, Debug)]
-pub enum LinkatFlags {
-    SymlinkFollow,
-    NoSymlinkFollow,
+libc_bitflags! {
```

Comment:
This type itself should be `#[cfg(not(target_os = "redox"))]` as the file system on redox does not support symlink

Reply:
Actually, I am not sure if we should define a new type or just use the [`AtFlags`](https://docs.rs/nix/latest/nix/fcntl/struct.AtFlags.html) type.

It is safer to define a new type so that we can prevent users from passing invalid arguments, but this will add our maintainence burgen. We are already using `AtFlags` type for other functions like [`fstatat`](https://docs.rs/nix/latest/nix/sys/stat/fn.fstatat.html), cc @asomers, what do you think?

Reply:
Link [Add EmptyPath for FchownatFlags](https://github.com/nix-rust/nix/pull/1128) as these 2 PRs are similar